client/src/components/event-card/EventCard.jsx

```diff
@@ -90,16 +90,18 @@ function EventCard({
     };
     if (endDateTime < startDateTime)
       alert("End Time could not be before Start Time");
```

Comment:
I think it should alert something like "You can not set the end time before the start time" 

Other than that, looks good.

Reply:
fixed